synapse/lib/stormlib/cache.py

```diff
@@ -173,6 +173,7 @@ async def _runCallback(self, key):
             except s_stormctrl.StormReturn as e:
                 return await s_stormtypes.toprim(e.item)
             except s_stormctrl.StormCtrlFlow:
+                # DISCUSS / FIXME: This should probably convert the CtrlFlow into a StormRuntimeError?
```

Comment:
Seems like it, other control flow exceptions making it here would be an invalid callback query.

Reply:
I'll bring this up as a goback during scrum.